spec/models/miq_provision_workflow_spec.rb

```diff
@@ -130,7 +130,7 @@
   context '.class_for_platform' do
     {
       "openstack" => ManageIQ::Providers::Openstack::CloudManager::ProvisionWorkflow,
-      "redhat"    => ManageIQ::Providers::Redhat::InfraManager::ProvisionWorkflow,
+      "ovirt"     => ManageIQ::Providers::Ovirt::InfraManager::ProvisionWorkflow,
       "vmware"    => ManageIQ::Providers::Vmware::InfraManager::ProvisionWorkflow,
     }.each do |k, v|
       it(k) { expect(described_class.class_for_platform(k)).to eq(v) }
```

Comment:
This looks like one of those specs that shouldn't be in core or at least not in this way since it references specific provider plugins

Reply:
I think the problem is there's no way to test if it works if everything is in plugins.  Or rather, it requires every provider author to write a test, which is clumsy and forgettable. Maybe we can find a way to do it in a providers_common kind of way, but I can't see how.

Another option is to do our dummy provider thing, where we include it in dev/test, and then write core specs against the dummy provider.

Reply:
Yeah we could move this to a shared example and check that the passed in platform results in the described_class then include that in the plugin provision_workflow_specs